twisted/web/_http2.py

```diff
@@ -625,7 +625,8 @@ def _handleWindowUpdate(self, event):
                 # unnecessary but benign. We'll ignore it.
                 return
 
-            self.priority.unblock(streamID)
+            if self._outboundStreamQueues.get(streamID):
```

Comment:
We cannot. `priority.unblock()` doesn't raise an `IndexError` at this point in the code, because the stream *is* in the priority tree. Instead it raises an `IndexError` much later in the prioritised data sending loop, where we attempt to send some data we don't actually have.